ComponentKitApplicationTests/CKStackLayoutComponentTests.mm

```diff
@@ -681,6 +740,40 @@ - (void)testPositiveViolationIsDistributedEquallyAmongGrowingAndShrinkingFlexibl
   CKSnapshotVerifyComponent(c, kSize, nil);
 }
 
+- (void)testPositiveViolationIsDistributedEquallyAmongGrowingAndShrinkingFlexibleChildComponentsExpressedAsPercentages
```

Comment:
nit: How about `*ArbitraryFloats`? Since the flex factors in this test isn't really adding up to 1.